src/Image.js

```diff
@@ -93,7 +93,8 @@ function ImageBase({
       {...imgProps}
       ref={sizeRef}
       sizes={sizes || `${size}px`}
-      {...{ className, src, srcSet, width, height, style }}
+      srcSet={(sizes || size > 1) ? srcSet : thumb }
```

Comment:
Zou dit niet `&&` moeten zijn? Als je maar 1 van de twee hebt is het niet handig om de `srcSet` te zetten toch?

Reply:
Als sizes bestaat, heeft de gebruiker die opgegeven (want anders zou `${size}px` gebruikt worden). Er is in dat geval al bij de server side render een `srcSet` nodig.

Als sizes niet bestaat, wordt `${size}px` gebruikt als waarde voor `sizes`. De default waarde daarvan is 1, pas na mount kan de echte size worden gemeten en zal de value dus groter dan 1 zijn. In dat geval is dan pas de `srcSet` nodig.

Reply:
Yes, sorry. Ik had even niet door dat `sizes` en `size` allebei in `sizes` terecht kwamen.